libraries/core-react/src/components/TextField/Field.tsx

```diff
@@ -170,45 +186,30 @@ export const Field = forwardRef<
   }
 
   return (
-    <>
-      {inputIcon || unit ? (
```

Comment:
This condition was the reason for the re-render of input when error handling was used